ui/src/components/DashboardServices.js

```diff
@@ -0,0 +1,295 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import { Tooltip, StatusText } from '@scality/core-ui';
+import {
+  spacing,
+  fontSize,
+  fontWeight,
+} from '@scality/core-ui/dist/style/theme';
+
+import type { Alert } from '../services/alertUtils';
+import type { Status } from '../containers/AlertProvider';
+import { PageSubtitle } from '../components/style/CommonLayoutStyle';
+import {
+  useAlertLibrary,
+  useHighestSeverityAlerts,
+  highestAlertToStatus,
+} from '../containers/AlertProvider';
+import CircleStatus from './CircleStatus';
+import { STATUS_HEALTH } from '../constants.js';
+import { formatDateToMid1 } from '../services/utils';
+
+const ServiceItems = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: ${spacing.sp4};
+`;
+
+const ServiceItemLabelWrapper = styled.div`
+  display: flex;
+  align-items: baseline;
+`;
+
+const ServiceItemLabel = styled.div`
+  margin-left: ${spacing.sp8};
+`;
+
+const ServiceItemElement = styled.div`
+  padding: ${spacing.sp4};
+`;
+
+const ClickableServiceItemElement = styled(ServiceItemElement)`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  :hover {
+    background-color: ${(props) => props.theme.highlight};
+  }
+`;
+
+const NonHealthyServiceItemElement = styled.div`
+  cursor: pointer;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+
+  a {
+    text-decoration: none;
+    color: inherit;
+    width: 100%;
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const NonHealthyPopUp = styled.div`
+  display: flex;
+  flex-direction: column;
+  font-size: ${fontSize.base};
+
+  label {
+    width: 25%;
+    margin-right: ${spacing.sp8};
+    color: ${(props) => props.theme.textSecondary};
+    text-align: right;
+  }
+`;
+
+const NonHealthyPopUpTitle = styled.div`
+  font-weight: ${fontWeight.bold}
+  text-align: center;
+`;
+
+const NonHealthyPopUpItem = styled.div`
+  width: 100%;
+  display: flex;
+  margin: ${spacing.sp4} ${spacing.sp14};
+  align-items: center;
+`;
+
+const ClickableIcon = styled.i`
+  self-align: flex-end;
+`;
+
+const ServiceItem = ({
+  label,
+  status,
+  alerts,
+}: {
+  label: string,
+  status: Status,
+  alerts: Alert[],
+}) => {
+  let date = null;
+  const intl = useIntl();
+
+  const getStartDate = (alerts: Alert[]) => {
+    // Check we only have alerts and set startsAt properties
+    const filtered = alerts.filter((item) => item && item.startsAt);
```

Comment:
I think the filter is not necessary here, since what you want is only a check to make sure we do have the startsAt property.